package.json

```diff
@@ -10,6 +10,7 @@
     "@testing-library/user-event": "^7.1.2",
     "clsx": "^1.1.1",
     "react": "^16.13.1",
+    "react-burger-menu": "^3.0.3",
```

Comment:
MUI does offer a sidebar component. Though this one does look very cool, it might be better to use the MUI one for consistency. 

If you think using this is necessary (or you really can't be bothered changing), the colors/theming need a lot of changes. (have to test on dark mode too)

Reply:
Change the whole sidebar component to using only MUI